src/phpFastCache/Drivers/Zendshm/Driver.php

```diff
@@ -130,11 +130,20 @@ protected function driverConnect()
      */
     public function getStats()
     {
-        $stats = (array) zend_shm_cache_info();
-        return (new driverStatistic())
-            ->setData(implode(', ', array_keys($this->namespaces)))
```

Comment:
Like Scrutinizer [said](https://scrutinizer-ci.com/g/PHPSocialNetwork/phpfastcache/inspections/eb8f658f-5772-4522-9449-fcccd7dc4eed/issues/files/src/phpFastCache/Drivers/Zendshm/Driver.php?status=new&orderField=path&order=asc&honorSelectedPaths=0):

I think that you made a mistake here: The property `$this->namespaces` does not exists, you may want to use `$this->itemInstances` instead ?